khp.md

```diff
@@ -383,10 +381,17 @@ Mechanism to handle storing evolution conditions
 
     syntax KItem ::= "#wolfram.quantifierElimination" "(" FullFormExpression ")"    [function]
 
-    rule #wolfram.quantifierElimination( WLFRAMEXPR )
+    syntax Ids ::= "List2Ids" "(" List ")" [function]
+    rule List2Ids(.List) => .Ids
+    rule List2Ids(ListItem(I:Id) L:List) => I , List2Ids(L)
+
+    rule <k> #wolfram.quantifierElimination( WLFRAMEXPR )
```

Comment:
If I don't specify the K cell in this rule, I can build and run tests. For some reason, when I add the K cell (both as is, and with adding `...` at the end) I run into non-termination issues